cedar-policy/CHANGELOG.md

```diff
@@ -42,65 +46,6 @@ validator will return an error for a policy that can never fire.
 
 ## 2.3.2
 
-### Removed
-- Move public API for partial evaluation behind experimental feature flag. To
-  continue using this feature you must enable the `partial-eval` feature flag.
-- Export the `cedar_policy_core::evaluator::{EvaluationError, EvaluationErrorKind}` and
```

Comment:
This large block of deleted text was duplicate from elsewhere in the file. Likely the result of some merging issue.